lib/pure/collections/tables.nim

```diff
@@ -771,6 +771,14 @@ proc del*[A, B](t: var OrderedTableRef[A, B], key: A) =
   ## deletes `key` from ordered hash table `t`. O(n) comlexity.
   t[].del(key)
 
+proc `==`*[A, B](s, t: OrderedTable[A, B]): bool =
+  equalsImpl()
```

Comment:
This is not correct, 2 ordered tables are equal if and only if the insertion order is also identical.